### PR TITLE
Fix invalid weapons (swatequipment.)

### PR DESCRIPTION
**Pull request by AlfieLC**
BadguyColt1911HG -> BadguyColtM1911HG
FNFAL -> FNFALMG

### CODE REVIEW

File: System/EnemyArchetypes.ini [hunk 1]
```diff
@@ -2255,7 +2255,7 @@ PrimaryWeapon=(Weapon="SwatEquipment.SG552MG",Chance=14)
 PrimaryWeapon=(Weapon="SwatEquipment.M1Super90SG",Chance=14)
 PrimaryWeapon=(Weapon="SwatEquipment.Pump2RifleGripSG",Chance=10)
 PrimaryWeapon=(Weapon="SwatEquipment.ColtSMG",Chance=2)
-PrimaryWeapon=(Weapon="SwatEquipment.FNFAL",Chance=4)
+PrimaryWeapon=(Weapon="SwatEquipment.FNFALMG",Chance=4)
 BackupWeapon=(Weapon="SwatEquipment.USP40HG",Chance=33)
```
Comment (T-MaxWiese-T) [inline System/EnemyArchetypes.ini:2259]: what about this? this weapon doesn't even exist

File: System/EnemyArchetypes.ini [hunk 2]
```diff
@@ -2160,7 +2160,7 @@ BackupWeapon=(Weapon="SwatEquipment.Glock9mmHG",Chance=20)
 BackupWeapon=(Weapon="SwatEquipment.Glock19HG",Chance=20)
 BackupWeapon=(Weapon="SwatEquipment.BadguyColtM1911HG",Chance=20)
 BackupWeapon=(Weapon="",Chance=20)
```
Comment (T-MaxWiese-T) [inline System/EnemyArchetypes.ini:2162]: what about this? here is no weapon at all

File: System/EnemyArchetypes.ini [hunk 3]
```diff
@@ -2160,7 +2160,7 @@ BackupWeapon=(Weapon="SwatEquipment.Glock9mmHG",Chance=20)
...
-BackupWeapon=(Weapon="BrowningHPHG",Chance=20)
+BackupWeapon=(Weapon="SwatEquipment.BrowningHPHG",Chance=20)
 Equipment1=(Equipment="SwatSimpleEquipment.Glasses_a",Chance=30)
 Equipment1=(Equipment="SwatSimpleEquipment.Glasses_c",Chance=30)
 Equipment1=(Equipment="",Chance=40)
```
Comment (T-MaxWiese-T) [inline System/EnemyArchetypes.ini:2166]: what about this? here is no weapon at all

File: System/EnemyArchetypes.ini [hunk 4]
```diff
@@ -2195,7 +2195,7 @@ PrimaryWeapon=(Weapon="SwatEquipment.M1Super90SG",Chance=10)
 PrimaryWeapon=(Weapon="SwatEquipment.Beretta92FSHG",Chance=10)
 PrimaryWeapon=(Weapon="SwatEquipment.BadguyColtM1911HG",Chance=5)
 PrimaryWeapon=(Weapon="",Chance=5)
```
Comment (T-MaxWiese-T) [inline System/EnemyArchetypes.ini:2197]: what about this? here is no weapon at all

**Review by T-MaxWiese-T — DISMISSED** [review]

**Comment by T-MaxWiese-T** [comment]
there are 12 places where nothing is written

File: System/EnemyArchetypes.ini (hunk 1 above)
Comment (AlfieLC) [reply System/EnemyArchetypes.ini:2259]: It does in the code.

File: System/EnemyArchetypes.ini (hunk 2 above)
Comment (AlfieLC) [reply System/EnemyArchetypes.ini:2162]: Yes, there is a chance for no secondary weapon.

File: System/EnemyArchetypes.ini (hunk 3 above)
Comment (AlfieLC) [reply System/EnemyArchetypes.ini:2166]: Meaning there is a possibility for no backup weapon.

File: System/EnemyArchetypes.ini (hunk 4 above)
Comment (AlfieLC) [reply System/EnemyArchetypes.ini:2197]: 5% chance for no secondary.

**Pull request merged**